src/components/events/EventForm.jsx

```diff
@@ -46,7 +53,7 @@ class EventForm extends React.Component {
     const { eventDate, eventTitle, numJudges } = this.state;
     const collectionName = 'events';
     const data = {
-      eventDate: new Date(eventDate),
+      eventDate: new Date(parseInt(eventDate.substring(6, 10)), parseInt(eventDate.substring(3, 5)) - 1, parseInt(eventDate.substring(0, 2)), 4, 0, 0),
```

Comment:
can the constructor for date accept a locale string?

Reply:
Yes, the date constructor can accept a string. There was a problem where the date was assumed to be EDT when passed in and when the date object was translated into a string, it would produce UTC which is four hours behind. So if I passed in 20/02/2000, then the date objects would actually be 4 hours behind of 20/02/2000 00:00 EDT and the date represented would be 19/20/2000 17:00 UTC. I fixed this by adding 4 hours to 00:00 EDT so that when the date is converted to UTC, the date represented would still be the same intended date. 

But yes, the solution is very inelegant. I think I found a better solution for this and hopefully it works.